packages/web-twig/src/Resources/components/UNSTABLE_EmptyState/__tests__/__fixtures__/UNSTABLE_EmptyState.twig

```diff
@@ -1,12 +1,15 @@
 <!-- Render with all props -->
-<UNSTABLE_EmptyState
-    buttonPrimaryText="Action"
-    buttonPrimaryUrl="#"
-    buttonSecondaryText="Action"
-    buttonSecondaryUrl="#"
-    description="Look somewhere else"
-    iconName="search"
-    linkText="Link to something"
-    linkUrl="#"
-    title="Empty State Title"
-/>
+<UNSTABLE_EmptyState spacing="space-700">
```

Comment:
Ok, nice idea :-) Let's discuss this and there is no right or wrong answer.

From the point of a developer who will be using this, I can now do:

```jsx
<EmptyState>
  <Heading>Heading</Heading>
  <Text>This is an empty state</Text>
  <Link href="/">Back to homepage</Link>
</EmptyState>

// or 

<EmptyState>
  <EmptyStateContent
        description="This is an empty state"
        heading="Heading"
    />
  <EmptyStateButtons
        buttonPrimaryText="Back to homepage"
        buttonPrimaryUrl="/"
    />
</EmptyState>

// what about the third option? are we letting it go?

<EmptyState
  description="This is an empty state"
   heading="Heading"
   buttonPrimaryText="Back to homepage"
   buttonPrimaryUrl="/"
/>
```

I am not suggesting that we must do this right now. I am just trying to find the point where we should stop our iterations and merge this and let the developer decide about the ease of use.

There could also be another option, that also makes sense. However the naming of the subcomponent EmptyStateButtons make no sense here:

```jsx
<EmptyState>
  <EmptyStateContent>
    <Heading>Heading</Heading>
    <Text>This is an empty state</Text>
  </EmptyStateContent>
  <EmptyStateButtons>
    <Link href="/">Back to homepage</Link>
  </EmptyStateButtons>
</EmptyState>
```

As I wrote at the beginning, there is no right or wrong, just the ease of using the API we fabricate. So maybe just let it be as it is right now and test if the usability is good enough :-)

Reply:
According to discussion it will be:

```
<EmptyState>
    <EmptyStateSection>
        <Icon name="icon-name" />
    </EmptyStateSection>
    <EmptyStateSection>
        <Heading>Heading</Heading>
        <Text>This is an empty state</Text>
    </EmptyStateSection>
    <EmptyStateSection>
        <ActionsLayout>
            <ButtonLink href="/" color="primary">Back to homepage</ButtonLink>
            <ButtonLink href="/" color="primary">Back to homepage</ButtonLink>
        </ActionsLayout>
    </EmptyStateSection>
    <EmptyStateSection>
        <Link href="/">Back to homepage</Link>
    </EmptyStateSection>
</EmptyState>
```

Reply:
I think we are able to make the structure a lot simpler:

```jsx
<EmptyState>
    <Icon name="icon-name" />
    <Heading>Heading</Heading>
    <Text>This is an empty state</Text>
    <ActionsLayout>
        <ButtonLink href="/" color="primary">Back to homepage</ButtonLink>
        <ButtonLink href="/" color="primary">Back to homepage</ButtonLink>
    </ActionsLayout>
    <Link href="/">Back to homepage</Link>
</EmptyState>
```

It just requires something like the following CSS:

```scss
.UNSTABLE_EmptyState {
  // …

  max-width: 582px; // if this is the desired limit for the component and not just the demo
  justify-items: center;
}
```

What do you think? Or did I miss anything?

Reply:
Based on the design, there is some space between the sections. That is why we use the subcomponent. Please, look at the Figma file. Maybe we missed something but the offset between the sections is different than between components themselves.

Reply:
@curdaj I'm totally OK with what you suggest. You would use CSS to add a gap among anything inside `EmptyState`, plus the syntax of the whole `EmptyState` composition is much simpler.

Reply:
@adamkudrna Based on discussion with @literat , we decided to keep EmptyStateSection for now. We consider it as an unstable component and may change it in the future